upf/upf_cli.c

```diff
@@ -1147,28 +1159,45 @@ upf_show_session_command_fn (vlib_main_t * vm,
       unformat_free (line_input);
     }
 
-#if FLOWTABLE_TODO
+  if (has_flows && !has_up_seid)
+    {
+      error =
+	clib_error_return (0, "must specify UP F-SEID to show session flows");
+      goto done;
+    }
+
   if (has_flows)
     {
+      u32 cpu_index;
+      flowtable_main_t *fm = &flowtable_main;
+      vlib_thread_main_t *tm = vlib_get_thread_main ();
+      flows_out_arg_t arg = {.vm = vm,.seid = up_seid,.filtered =
+	  true,.limit = limit
+      };
```

Comment:
Strange that this passes through the format checks, I would expected that there should be a space after a comma.

Reply:
Hmm, it looks like it was actually `indent` which formatted it this way, and I overlooked it. Split it into separate lines